manim/mobject/types/vectorized_mobject.py

```diff
@@ -1364,9 +1364,12 @@ def get_arc_length(self, sample_points_per_curve: Optional[int] = None) -> float
 
     # Alignment
     def align_points(self, vmobject):
-        # This probably makes the current vmobject and the given one have the same number of points,
-        # by adding extra points to the last sub-path. This method is never used in the whole library.
+        """Adds points to self and vmobject so that they both have the same number of subpaths, each of the matching lengths.
+        Points are added either by subdividing curves evenly along the subpath, or by creating new subpaths consisting
+        of a single point repeated.
+        """
```

Comment:
**Question :** Why« matching lengths »? I don't see where the notion of length comes from, since it's just adding dummy curves at each subpath, but I'm probably missing something.

Reply:
Uh, length of the subpath as in number of curves in that sub path. I'm open to better phrasing.

Reply:
I see, thanks for the clarification. Other functions use the notion of _length_ (in the sense of a "norm"), so this should be reformulated. 

Why not something like : "Adds points to self and vmobject so that they both have the same number of subpaths, and that for each subpath of `self` the corresponding subpath in `vmobject` will have the same number of curves."

I'm not a native english speaker, so I'm not the best at doing good formulation :/